libraries/ms-helper/inc/sequenced_relay.h

```diff
@@ -1,26 +1,31 @@
 #pragma once
-// Handles motor relay CAN requests and sequencing
+// Handles relay CAN requests and sequencing
 // Requires CAN, soft timers to be initialized.
 //
 // Sequences relay closing with a slight delay to offset the high make current of the HV relay coil.
 // We also assume the relays are active-high.
+#include "exported_enums.h"
 #include "gpio.h"
 #include "relay_rx.h"
 #include "soft_timer.h"
 #include "status.h"
 
-typedef struct MotorRelaySettings {
+typedef struct SequencedRelaySettings {
+  SystemCanMessage can_message;
```

Comment:
Are we using CANMessageId or SystemCanMessage? I've used both previously...

Reply:
They should be equivalent? `CANMessageId` is probably more proper from a library perspective since otherwise `ms-helper` should be including `codegen-tooling`.